.github/workflows/sival_nightly.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Nigthly
+on:
+  schedule:
+    - cron: "00 04 * * *"
+
+jobs:
+  FPGA-CW310-SiVal-Nightly:
+    name: FPGA CW310 SiVal nightly tests
```

Comment:
nit: you don't need to repeat "nightly tests" since it will already show as "Nightly / FPGA CW310 SiVal"